### ISSUE TITLE
How to get a version in a language other than English

### ISSUE THREAD

**Issue by zwiastunsw** [IBMa/equal-access#459]
I trying to create a browser version of the tester locally. 
I have done translations of all the content that should be translated (help files, error definition file, error mapping file and WCAG. After compilation I get a working version, but - unfortunately - some of the reports are in English, not Polish. This is the case with both HTML and  XLSX reports  
![ibm](https://user-images.githubusercontent.com/29527200/123558227-6afd0d80-d795-11eb-8929-3bc91ee2399c.jpg)

Two elements are not translated: description of the problem and references to WCAG

**Comment by tombrunet** [comment]
The messages come from the engine: https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v2/checker/accessibility/nls/index.ts

At the moment, we only have english support, but with the strings separated, may not be a major effort to add in locale/language detection and switch it. If you want to run locally, you can edit the strings in the file and run a local rule server (Instructions at https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/README-RULES.md#run-local-server-with-local-browser-extension).

If you have a fork with the translations, might help us have a sample that we can work with to speed that along.

**Comment by zwiastunsw** [comment]
Yes, I  have a fork with translation: https://github.com/lepszyweb/equal-access (branch polish-version)
I translated all files in directory 
accessibility-checker-engine\src\v2\checker\accessibility\nls and accessibility-checker-engine\src\v2\checker\accessibility\ruleset.
in my opinion, it should work. I don't understand why it doesn't work.

**Comment by zwiastunsw** [comment]
Yes, yes, yes! Its works! thx

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/459?planning-poker) with ZenHub @aliunwala @drjoho @mbgower @philljenkins @shunguoy @tombrunet

**Comment by MHoov** [comment]
Consider capturing the info above to put on README for folks to do translation on their own - with clause about not guaranteeing accuracy, etc.
